src/caffe/layers/detectnet_transform_layer.cpp

```diff
@@ -81,7 +81,7 @@ void DetectNetTransformationLayer<Dtype>::retrieveMeanImage(Size dimensions) {
 
   // resize, if dimensions were defined:
   if (dimensions.area() > 0) {
-    resize(data_mean_, data_mean_, dimensions, cv::INTER_CUBIC);
+    resize(data_mean_, data_mean_, dimensions);
```

Comment:
I was thinking it doesn't matter much because the mean image size will almost always match the incoming data size. But you're right - I might as well follow what @drendleman had originally intended.